mozci/push.py

```diff
@@ -1151,7 +1151,10 @@ def _map_failing_tasks(groups):
         )
 
     def classify(
-        self, confidence_medium: float = 0.8, confidence_high: float = 0.9
+        self,
+        confidence_medium: float = 0.8,
+        confidence_high: float = 0.9,
+        should_retrigger: bool = False,
```

Comment:
Even if we implement the retriggering as a flag of the classify command, I'd write two separate functions, `classify` and `retrigger`.
`retrigger` could accept a list (or iterable) of tasks as input, and the caller will first call `classify` and then call `retrigger` with the list (or iterable) of unknown tasks.

Reply:
okay